File: src/main/java/com/github/gossie/paddysbot/PaddysBotController.java
package com.github.gossie.paddysbot;

import com.slack.api.bolt.App;
import com.slack.api.bolt.servlet.SlackAppServlet;

import javax.servlet.annotation.WebServlet;

@WebServlet("/slack/events")
public class PaddysBotController extends SlackAppServlet {

    PaddysBotController(final App app) {
        super(app);
    }

}

File: src/main/kotlin/com/github/gossie/paddysbot/PaddysBotController.kt
package com.github.gossie.paddysbot

import com.slack.api.bolt.App
import com.slack.api.bolt.servlet.SlackAppServlet

import javax.servlet.annotation.WebServlet

@WebServlet("/slack/events")
class PaddysBotController(app: App) : SlackAppServlet(app)
